docs/modules/ROOT/pages/api-ref.adoc

```diff
@@ -3,9 +3,15 @@
 
 A reference guide to the Hazelcast Platform Operator CRD types.
 
+#Authors note - is this Operator API docs or custom resource documentation? Which is it?#
```

Comment:
It's neither an Operator API doc nor a custom resource. It's more like a guide how to migrate to the Operator for the users of Helm chart